api/tests/Api/Activities/ReadActivityTest.php

```diff
@@ -93,7 +93,7 @@ public function testGetSingleActivityIsAllowedForMember() {
         $this->assertEquals($this->getIriFor($activity->getRootContentNode()), $data['_embedded']['rootContentNode']['_links']['self']['href']);
         $this->assertEquals($this->getIriFor($activity->getRootContentNode()), $data['_embedded']['rootContentNode']['_links']['root']['href']);
         $this->assertContains(['href' => $this->getIriFor('multiSelect1')], $data['_embedded']['rootContentNode']['_links']['children']);
-        $this->assertEquals($this->getIriFor('materialNode1'), $data['_embedded']['contentNodes'][0]['_links']['self']['href']);
```

Comment:
Testing for the first item if flaky, because it depends on the sorting of the embedded items. At the moment, sorting of embedded items is undefined (https://github.com/ecamp/ecamp3/issues/2651). Hence switching to count of embedded items.